frontend/__tests__/routes/_protected/applications/index.test.tsx

```diff
@@ -124,31 +124,31 @@ vi.mock('~/services/benefit-application-service.server', () => ({
             dentalInsurance: true,
             personalInformation: {
               copyMailingAddress: false,
-              homeAddress: '3102 Main St, BC',
+              homeAddress: '567 Home Addr',
               homeApartment: 'Unit #302',
-              homeCity: 'Vancouver',
+              homeCity: 'BC City',
               homeCountry: '0cf5389e-97ae-eb11-8236-000d3af4bfc3',
-              homePostalCode: 'V5T 3G7',
+              homePostalCode: 'J8T 7X7',
               homeProvince: '9c440baa-35b3-eb11-8236-0022486d8d5f',
-              mailingAddress: '20399 Douglas Cres',
+              mailingAddress: '789 Mailing Addr',
               mailingApartment: '',
-              mailingCity: 'Langley',
+              mailingCity: 'MailingCity',
               mailingCountry: '0cf5389e-97ae-eb11-8236-000d3af4bfc3',
-              mailingPostalCode: 'V3A 4B3',
+              mailingPostalCode: 'J8T 7X7',
               mailingProvince: '9c440baa-35b3-eb11-8236-0022486d8d5f',
-              phoneNumber: '+1 604 514 2800',
-              phoneNumberAlt: '+1 604 638 4852',
+              phoneNumber: '+1 555 514 2800',
+              phoneNumberAlt: '+1 703 701 8991',
```

Comment:
This appears to be a valid phone number registered to *Usa Mobility Wireless*

Reply:
Fixed: 9323cab4f704052c540a8799beb30a2f0a651956

Reply:
Phone number still appears to be a valid phone number. Consider using the phone numbers that are in the examples for the [libphonenumber-js](https://www.npmjs.com/package/libphonenumber-js) library.